astroplan/scheduling.py

```diff
@@ -350,22 +559,21 @@ def from_timespan(cls, center_time, duration, **kwargs):
         duration : `~astropy.units.Quantity` or `~astropy.time.TimeDelta`
             Duration of time-span to schedule
         """
-        start_time = center_time - duration/2.
-        end_time = center_time + duration/2.
+        start_time = center_time - duration / 2.
+        end_time = center_time + duration / 2.
         return cls(start_time, end_time, **kwargs)
 
     def _make_schedule(self, blocks):
-
         # Combine individual constraints with global constraints, and
         # retrieve priorities from each block to define scheduling order
         _all_times = []
         _block_priorities = np.zeros(len(blocks))
-        for i,b in enumerate(blocks):
+        for i, b in enumerate(blocks):
             if b.constraints is None:
                 b._all_constraints = self.constraints
             else:
                 b._all_constraints = self.constraints + b.constraints
-            b._duration_offsets = u.Quantity([0*u.second, b.duration/2, b.duration])
+            b._duration_offsets = u.Quantity([0 * u.second, b.duration / 2, b.duration])
```

Comment:
I wonder if this should be a property of the ```Constraint``` itself - how it is to be evaluated over an ```ObservingBlock```'s duration? 

I raise this suggestion partly as a result of the issues I've noticed with the ```MeridianTransitConstraint``` in #165. The way that's written at the moment, a user would only care if that constraint was valid at the start of the ```ObservingBlock```.

It may be possible to re-write that particular constraint, but perhaps there is a case that some constraints need to be evaluated on a fine time grid, others on a coarse time grid, and others just at the start of the ```ObservingBlock```?